src/SideMenu.tsx

```diff
@@ -133,7 +135,11 @@ export const SideMenu = memo(
                                             {"items" in item ? (
                                                 <>
                                                     <button
-                                                        aria-expanded="false"
+                                                        aria-expanded={
+                                                            item.expandedByDefault
```

Comment:
```suggestion
                                                            (item.expandedByDefault ?? false)
```

Reply:
We dont allow using variables that are not boolean as if they where boolean. It's sources of error, empty string being falsy, 0 beening falsy, default being implicitly false.